platform-ui-skin/pom.xml

```diff
@@ -37,6 +37,7 @@
             <plugin>
                 <groupId>org.lesscss</groupId>
                 <artifactId>lesscss-maven-plugin</artifactId>
+                <version>1.7.0.1.1</version>

```

Comment:
-> Maven plugins version must not be defined directly in modules pom, they are already defined by the parent pom.

Yes, but ->   

  <groupId>org.exoplatform</groupId>
  <artifactId>maven-parent-pom</artifactId>
  <version>16-SNAPSHOT</version>

is not part of the platform-ui project ?

So I did this ->

       <plugin>
          <groupId>org.lesscss</groupId>
          <artifactId>lesscss-maven-plugin</artifactId>
          <version>${version.lesscss.plugin}</version>
        </plugin>

but I could do this ->

    <pluginManagement>
      <plugins>
        <plugin>
          <groupId>org.apache.maven.plugins</groupId>
          <artifactId>maven-war-plugin</artifactId>
          <configuration>
            <!-- Don't include libraries into Web Archives -->
            <packagingExcludes>WEB-INF/lib/*.jar</packagingExcludes>
          </configuration>
        </plugin>
        <plugin>
          <groupId>org.lesscss</groupId>
          <artifactId>lesscss-maven-plugin</artifactId>
          <version>1.7.0.1.1</version>
        </plugin>
      </plugins>
    </pluginManagement>